CoreScriptsRoot/LoadingScript.lua

```diff
@@ -5,89 +5,82 @@
 -- Constants
 local PLACEID = game.PlaceId
 
-local MPS = game:GetService('MarketplaceService')
-local UIS = game:GetService('UserInputService')
-local guiService = game:GetService("GuiService")
+local MarketplaceService = game:GetService('MarketplaceService')
+local InputService = game:GetService('UserInputService')
```

Comment:
In the future, can we name these locals after the service without shortening them? I know it's annoying, but it reduces ambiguity and I'd like to eventually make it all consistent!